### ISSUE TITLE
Passwords and env settings

### ISSUE THREAD

**Issue by tcharlat** [sbres/Docker_seedbox42#3]
Passwords and/or usernames can be forwarded to running docker container when up'ing the docker-compose.yml
Either by passing env or from a config file

**Comment by sbres** [comment]
Yes, It's on the build on the file mongodb/set_mongodb_password.sh

I removed all breaking points, I might add the env variables in the compose file.

**Comment by sbres** [comment]
Do you have a easy way to add a user, like curl a url or something.
I found it's quite hard to make the bcrypt using only bash.
